src/api/app/models/bs_request.rb

```diff
@@ -813,13 +813,14 @@ def setincident(incident)
   end
 
   def send_state_change
-    intermediate_state = ['new', 'review']
-    return if state_was.to_s == state.to_s
+    return unless state_changed?
     # new->review && review->new are not worth an event - it's just spam
-    return if state.to_s.in?(intermediate_state) && state_was.to_s.in?(intermediate_state)
+    return unless conclusive?
 
     options = event_parameters
-    options[:duration] = (updated_at - created_at).to_i if conclusive?
+
+    # measure duration unless superseding a final state, like revoked -> superseded
+    options[:duration] = (updated_at - created_at).to_i if FINAL_REQUEST_STATES.exclude?(state_was.to_sym) && FINAL_REQUEST_STATES.exclude?(state)
```

Comment:
@hennevogel the previous behaviour was `FINAL_REQUEST_STATES.exclude?(state_was.to_sym) && FINAL_REQUEST_STATES.include?(state)` is this change intended

Reply:
so a state change from `:new -> :accepted` would not record the duration

Reply:
Of course not. copy pasta :) Should be `exclude(was) && include(is)`

- `FINAL_REQUEST_STATES.exclude?(:new) => true`
- `FINAL_REQUEST_STATES.include?(:accepted) => true`

or 

- `FINAL_REQUEST_STATES.exclude?(:accepted) => false`
- `FINAL_REQUEST_STATES.include?(:superseded) => true`

Good catch!